src/main/resources/template/PK/survey.json

```diff
@@ -0,0 +1,257 @@
+      "text":"How many children ages 5 to 13 attend school?",
+      "description":null,
+      "sequenceNo":3,
+      "responseDatas":
+      [
+        {
+          "text":"A. Not at all",
+          "value":0,
+          "sequenceNo":1
+        },
+        {
+          "text":"B. All, or no children ages 5 to 13",
+          "value":5,
+          "sequenceNo":2
+        }
+      ]
+    },
+    {
+      "componentKey":null,
+      "key":"elementary-occupations",
+      "text":"How many household members work in elementary occupations (not senior officials, managers, professionals, technicians or associated professionals, clerks, salespeople, service or shop workers, skilled workers in agriculture or fishery, craft or trade workers, or plant/machinery operators)?",
+      "description":null,
+      "sequenceNo":4,
+      "responseDatas":
+      [
+        {
+          "text":"A. Two or more",
+          "value":0,
+          "sequenceNo":1
+        },
+        {
+          "text":"B. One",
+          "value":5,
+          "sequenceNo":2
+        },
+        {
+          "text":"C. None",
+          "value":12,
+          "sequenceNo":3
+        }
+      ]
+    },
+    {
+      "componentKey":null,
+      "key":"highest-education",
+      "text":"What is the highest grade completed by the female head/spouse?",
+      "description":null,
+      "sequenceNo":5,
+      "responseDatas":
+      [
+        {
+          "text":"A. Less than Class 1 or no data",
+          "value":0,
+          "sequenceNo":1
+        },
+        {
+          "text":"B. No female head/spouse",
+          "value":4,
+          "sequenceNo":2
+        },
+        {
+          "text":"C. Class 1 or higher",
+          "value":6,
+          "sequenceNo":3
+        }
+      ]
+    },
+    {
+      "componentKey":null,
+      "key":"drinking-water",
+      "text":"What is the main source of drinking water for the household?",
+      "description":null,
+      "sequenceNo":6,
+      "responseDatas":
+      [
+        {
+          "text":"A. Others",
+          "value":0,
+          "sequenceNo":1
+        },
+        {
+          "text":"B. Hand pump, covered/closed well, motorized pump/tube well, or piped water",
+          "value":3,
+          "sequenceNo":2
+        }
+      ]
+    },
+    {
+      "componentKey":null,
+      "key":"toilet-type",
+      "text":"What type of toilet is used by your household?",
+      "description":null,
+      "sequenceNo":7,
+      "responseDatas":
+      [
+        {
+          "text":"A. None or other",
+          "value":0,
+          "sequenceNo":1
+        },
+        {
+          "text":"B. Flush connected to pit/septic tank or open drain",
+          "value":2,
+          "sequenceNo":2
+        },
+        {
+          "text":"C. Flush connected to public sewerage",
+          "value":4,
+          "sequenceNo":3
+        }
+      ]
+    },
+    {
+      "componentKey":null,
+      "key":"refrigerator-freezer,

```

Comment:
You're missing a " here which is invalidating the JSON. @janicek1m  :)

Reply:
Fixed! Thank you!